docs/how-to-use-runhouse.rst

```diff
@@ -0,0 +1,256 @@
+
+For production settings, we recommend that users load cloud secrets, Kubeconfig, and available compute resources into Runhouse Den and authenticate from
+all launch environments using only the Runhouse token. Platform teams gain centralized observability over utilization, including insights into who is launching clusters,
+how often they are launched, and the resources or tasks executed on them. Access management becomes much simpler, especially in multi-cloud or multi-cluster environments.
+To get started with Den enabled, simply run ``runhouse login --sync-secrets`` in the CLI.
+
+.. image:: https://runhouse-tutorials.s3.amazonaws.com/Pull+Compute+from+Compute+Pool.jpg
+  :alt: Runhouse pulls compute from a pool of resources
+  :width: 600
+
+Start Your Project
+-------------------
+Once you have established access to compute, you can start developing a new ML project. The following steps will provide the details of how to use Runhouse, starting
+from a blank page in your IDE.
+
+1. Define Compute
+^^^^^^^^^^^^^^^^^
+Runhouse allows you to define compute requirements in code, and launch ephemeral clusters from the compute pool we described in the prior section.
+Here, you can define the required CPU, GPU, memory, and disk requirements (or name a specific cluster) to use.
+For instance, to create a cluster on AWS with an A10 GPU attached using an arbitrary Docker image, you can write:
+
+.. code:: python
+
+    import runhouse as rh
+
+    cluster = rh.ondemand_cluster(
+        name="rh-cluster", # This cluster can be saved and reused by name. We will prefix your username when saved, e.g. /my_username/rh-cluster
+        instance_type="A10G:1", # There are a number of options available for instance_type, check out the docs to see them all
+        provider="aws", # Specify a cloud provider
+        image_id="docker:nvcr.io/nvidia/pytorch:23.10-py3", # Use a Docker image
+        autostop_mins=90 # Remember to set autostop_mins to avoid leaving clusters running indefinitely.
+    ).up_if_not()
+
+You can run CLI commands on the cluster using ``cluster.run()`` to layer on setup steps beyond the underlying image; for instance, installing other packages.
+
+.. code:: python
+
+    cluster.run(['pip install numpy'])
+
+You can find full documentation about the Runhouse cluster API `in the Cluster docs <https://www.run.house/docs/tutorials/api-clusters>`_.
+
+1a. Starting the Runhouse Server Daemon
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+If not already running, the client will start the Runhouse API server daemon
+on the compute and form a secure network connection (either over SSH or HTTP/S).
+
+* The daemon can be thought of as a "Python object server," holding key-value pairs of names and Python objects in memory (objects you will dispatch to it in the next step), and exposing an HTTP API to call methods on those objects by name.
```

Comment:
nit: formatting seems to be a bit messed up here (at least on the gh previous), I think it's good practice to just generally include empty lines in between bullet points